src/AdminForm.php

```diff
@@ -2299,6 +2299,7 @@ public static function insertComponent(&$field, &$enabled, $settings, $create_fi
     if (isset($field['value_callback'])) {
       $method = 'get_default_' . $table . '_' . $name;
       $field['value'] = self::$method($field['form_key'], $options);
+      unset($field['value_callback']);
     }
```

Comment:
@jitendrapurohit - why do we need this here?

Reply:
@KarinG If we dont remove this here, it ends up in the yml `'#value_callback': true` where this property isn't supported. So you get something like this when you try to edit any element from the build page.


![image](https://user-images.githubusercontent.com/5929648/140604511-7aa99922-3607-4c71-9972-660ad5ba35d6.png)

The main requirement of this was to set a default value which i see is done just above this unset line 

    $field['value'] = self::$method($field['form_key'], $options);

Once that is done, i think we can unset it so it isn't stored in yml. Hope that helps :).